packages/limesuite/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=limesuite
+pkgver=v23.11.0.r19.g524cd2e
```

Comment:
you can edit the pkgver() like I did in https://github.com/BlackArch/blackarch/pull/4629/files#diff-1231f889678e893a8f0d23d15e31e3322f538cea94fe05b703b041ef817c1b9fR25 to remove the `v` prefix for projects that wrongly add `v` before the version in their git tags.

Reply:
Fixed